include/eve/arch/arm/sve/sve_true.hpp

```diff
@@ -13,42 +13,49 @@
 
 namespace eve::detail
 {
-// Returns clear sve_true for type
-// Calling svptrue_b8 is OK for most cases, where you OK with
-// 1s in insignificant bits but sometimes you are not.
-template<typename T>
-EVE_FORCEINLINE svbool_t
-sve_true()
-{
-  if constexpr( sizeof(T) == 1 ) return svptrue_b8();
-  else if constexpr( sizeof(T) == 2 ) return svptrue_b16();
-  else if constexpr( sizeof(T) == 4 ) return svptrue_b32();
-  else if constexpr( sizeof(T) == 8 ) return svptrue_b64();
-}
+  // Returns clear sve_true for type
+  // Calling svptrue_b8 is OK for most cases, where you OK with
+  // 1s in insignificant bits but sometimes you are not.
+  template<typename T>
+  EVE_FORCEINLINE svbool_t sve_true()
+  {
+    if constexpr( sizeof(T) == 1 ) return svptrue_b8();
+    else if constexpr( sizeof(T) == 2 ) return svptrue_b16();
+    else if constexpr( sizeof(T) == 4 ) return svptrue_b32();
+    else if constexpr( sizeof(T) == 8 ) return svptrue_b64();
+  }
 
-// Returns clear sve_true for type of a given cardinal
-// while masking potential garbage value
-template<relative_conditional_expr C, typename T>
-EVE_FORCEINLINE T
-sve_true(C cond, as<T> tgt)
-{
-  if constexpr(C::is_complete && C::is_inverted)
+  // Returns clear sve_true for type of a given cardinal
+  // while masking potential garbage value
+  template<relative_conditional_expr C, typename T>
+  EVE_FORCEINLINE T sve_true(C cond, as<T> tgt)
```

Comment:
there is ignore.mask(tgt) you don't need this

Reply:
We tested ignore.mask, leads to slightly worse codegen. We're going to specialize to_logical(relative) on SVE to use svwhilelt whenever possible.